Sphinx error building the portal site locally

<!-- Please include a self-contained copy-pastable example that generates the issue if possible.

Please be concise with code posted. See guidelines below on how to provide a good bug report:

- Craft Minimal Bug Reports http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
- Minimal Complete Verifiable Examples https://stackoverflow.com/help/mcve

Bug reports that follow these guidelines are easier to diagnose, and so are often handled much more quickly.
-->

**What happened**:

I tried to build the portal site locally on my Mac laptop while developing #67. I followed the procedure for this described on our [Contributor's Guide](https://projectpythia.org/pages/contributing.html#to-build-and-view-the-site-locally) to set up and activate a dedicated conda environment and then build the site using sphinx.

I got the following error:
```
Theme error:
An error happened in rendering the page index.
Reason: TypeError("generate_nav_html() missing 1 required positional argument: 'kind'")
make: *** [html] Error 2
```

**What you expected to happen**:

The site should build normally without error.

**Minimal Complete Verifiable Example**:

From repository root directory:
```bash
cd content
conda env create -f ../ci/environment.yml
conda activate pythia
make html
```

**Anything else we need to know?**:

**Environment**:

- Python version: 3.9.2
- Sphinx version: 3.5.4
- conda version: 4.9.2
- Operating System: macOS Catalina 10.15.7
- Install method (conda, pip, source): conda, as described above

--- a/ci/environment.yml
+++ b/ci/environment.yml
@@ -3,10 +3,8 @@ channels:
   - conda-forge
 dependencies:
   - myst-nb==0.10.1
-  - sphinx-panels
-  - pydata-sphinx-theme
-  - jupyterlab
-  - sphinx-autobuild
-  - matplotlib
   - sphinx-book-theme
+  - sphinx-panels
+  - matplotlib
   - pandas
+  - pyyaml

